sample_information_validator.py

```diff
@@ -32,14 +36,19 @@ class NumericValidator(object):
                        [default False]
     """
 
-    def validate(self, cell):
+    def validate_conc(self, cell):
         """Checks whether value is numeric or not."""
         logging.debug(cell.value, cell.data_type)
         if cell.data_type != 'n':
-            logger.error(
-                    'Cell {} is not numeric'.format(cell.coordinate)
+            try:
+                cell.value = float(cell.value.replace(",", ".")) # not sure how to change this in original file, would have to be done here
```

Comment:
Also, could there be valid thousands-',' if the user has a US-numeric setting in excel?